docs(AI): add AI development tools page and replace stale references (#4991)
patternfly/patternfly-org: packages/documentation-site/patternfly-docs/content/AI/marketplace/marketplace.md

---
id: AI development tools
section: AI
sortValue: 3
---

> 

Reviewer: ```suggestion
## What are plugins? 
```
Or, What is AI Helpers or What are PatternFly AI helpers - whatever direction we go

Author: I think this is only necessary if we don't use "AI development tools" for the record - since that's generic enough to where we don't need to explain what something is up-front. That's how this reads to me at least - clicking on a nav item but not knowing what it means - so we need to define it for them immediately.

Author: Updated as a part of the rename to "Marketplace"

## What's available

The marketplace is organized into plugins, each focused on a different area of PatternFly development. Here's what's currently available at a high level:

| Plugin | What it helps you do |
| :--- | :--- |
| **React** | Build and test React components following PatternFly coding standards and component structure best practices. |
| **Design foundations** | Work with design tokens, icons, colors, and spacing. Audit and validate token usage. |
| **Accessibility** | Audit and document accessibility across your project. |
| **Figma** | Review Figma design changes and generate code update checklists. |
| **Issue management** | Create, triage, and track issues across PatternFly repositories. Primarily for PatternFly maintainers. |
| **Repo management** | Manage releases, analyze dependencies, and maintain repository health. Primarily for PatternFly maintainers. |

Plugins are actively maintained and evolving. For the full, always-current list of every skill and agent available, see the [plugin catalog](https://github.com/patternfly/ai-helpers/blob/main/PLUGINS.md).

## Getting started

> 

Reviewer: ```suggestion
You can use our plugins with the AI coding tool of your choice, such as Claude Code or Cursor, by following these onboarding steps. 

For the best experience, set up the [PatternFly MCP server](https://github.com/patternfly/patternfly-mcp) to give AI tools direct access to PatternFly component documentation, prop schemas, and design guidelines. While our plugins work without the PatternFly MCP, connecting your tools to it will provide stronger results.
```

wdyt about moving the mcp note up here, to encourage the likelihood that people will use it from the get go? If you like it separate, we can just re-add the mcp section

### Claude Code

> 
> ```bash
> # Add the marketplace
> /plugin marketplace add patternfly/ai-helpers
> 
> # Install the plugins you need
> /plugin install react@ai-helpers
> ```

Reviewer: ```suggestion
To use our AI helpers in Claude Code: 

1. Add the AI helpers marketplace:
\```bash
/plugin marketplace add patternfly/ai-helpers
\```

2. Install the specific plugins you need:
\```bash
/plugin install react@ai-helpers
\```

3. Utilize the plugin's skills and knowledge in your project. 
```
wdyt about this format? fyi I just had to escape the backticks to avoid github formatting the code
